### PR TITLE
Fix min/max parameters in filter analysis

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: #11658

Now the min/max parameters are correctly initialized after selecting a column in a Filter analysis.

filter-analysis-parameters

For acceptance:
1. Add a Filter by column value analysis
2. Check than min/max values are 0/50
3. Click Apply

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-form.tpl [hunk 1]
```diff
@@ -28,8 +28,7 @@
       <% } %>
     </div>
   </div>
-  <% if (column) { %>
-  <div class="Editor-HeaderInfo">
+  <div class="Editor-HeaderInfo <% if (!column) { %>is-hidden<% } %>">
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-form.tpl]: I don't like this, if we don't want this dom element, then it should not be rendered. Are we showing it without rendering the whole view?

**Comment by nobuti** [comment]
How changing only the placeholder or hidding a node fixes the issue with the button? 🤔

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-form.tpl (hunk 1 above)
Comment (Jesus89) [reply lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-form.tpl]: The issue is that min/max default values are not rendered properly.

**Comment by Jesus89** [comment]
As soon as the default values min/max are properly set, the Apply button can be enabled and it works properly. The issue is that min/max values are not properly shown.

**Comment by Jesus89** [comment]
This commit added the issue with the min/max initial values: https://github.com/CartoDB/cartodb/commit/27c24531b21407de9758b7fe879a76d3cbe775d5. Without this changes, min/max values are rendered properly.

**Comment by nobuti** [comment]
What do you mean by not properly render? I can see the fields when the column is selected...

**Comment by Jesus89** [comment]
I mean the initial values (0, 50). This is the buggy behavior:
filter-analysis-wrong-min-max

**Comment by Jesus89** [comment]
These values: https://github.com/CartoDB/cartodb/blob/6af1042449cf440d9d2db7fb5f22e073b3cdb5d7/lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/filter-range-model.js#L20-L21

**Comment by nobuti** [comment]
I know which is the current behaviour, but the thing is that pattern is used in other forms, without an issue. I feel we need to dig deeper why this is happening, and why rendering (but hidden) those fields fixes it.

**Comment by Jesus89** [comment]
Thank you very much @nobuti for the explanation about the backbone-forms! :star2:
Now I see that if the element is not in the DOM the set fails and the values are set to null. Therefore, the logic must be implemented in the model to prevent this behavior.

**Comment by nobuti** [comment]
retest this please

**Comment by nobuti** [comment]
Sweet! 🚀

**Pull request merged**